lib/runtime-core/src/unique_id.rs

```diff
@@ -0,0 +1,78 @@
+//! Unique identifiers.
```

Comment:
I'd appreciate more of a description here too, about exactly what it is and why we'd use it.  But this is the lowest priority of the doc comments I'm asking for